tests/GameTest.elm

```diff
@@ -0,0 +1,34 @@
+module GameTest exposing (..)
+
+import Test exposing (..)
+import Expect
+
+import Board exposing (Mark(..), new)
+import Game exposing (..)
+
+all : Test
+all =
+    describe "Game Test"
+        [ test "A new game knows the current player is X" <|
+            \() ->
+              (Game.new (Board.new 9))
+                |> .currentPlayer
```

Comment:
Again, knows about the internal implementation of the datastructure for `Game`.
What about a function on `Game` that gives you the `currentPlayer`?